alibi_detect/base.py

```diff
@@ -94,6 +93,27 @@ def infer_threshold(self, X: np.ndarray) -> None:
         pass
 
 
+class DriftConfigMixin:
```

Comment:
Short docstring about what this class is for would be useful (same stuff as in the PR comments really).